plugins/outputs/awscsm/awscsm.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/aws/amazon-cloudwatch-agent/awscsm/csm"
 	"github.com/aws/amazon-cloudwatch-agent/awscsm/sdkmetricsdataplane"
 	"github.com/aws/amazon-cloudwatch-agent/awscsm/sdkmetricsdataplane/sdkmetricsdataplaneiface"
-	awsconfig "github.com/aws/amazon-cloudwatch-agent/cfg/aws"
```

Comment:
The existing code used the aliases `awsconfig`, `internalaws`, and `configaws` for this import.
I liked `configaws` the best, so I changed them all to be consistent.
Ideally we wouldn't need an alias, but `aws` is a pretty generic package name, who would have thought ;)